rqd/rqd/cuerqd.py

```diff
@@ -18,20 +18,20 @@
 """SYNOPSIS
      cuerqd [hostname] [OPTIONS]
       [hostname]            => RQD hostname (defaults to localhost)
-      -s                    => Print rqd status
-      -v                    => Print rqd version
-      --lp <cores>          => Lock Cores
-      --ulp <cores>         => Unlock Cores
-      --lh                  => Lock Host (All Cores)
-      --ulh                 => Unlock Host (All Cores)
-      --nimbyoff            => Turn on Nimby
-      --nimbyon             => Turn off Nimby
+      -s or --status        => Print rqd status
+      -v or --version       => Print rqd version
+      --lock <cores>        => Lock Cores
+      --unlock <cores>      => Unlock Cores
+      --lock all            => Lock Host (All Cores)
+      --unlock all          => Unlock Host (All Cores)
+      --nimby on            => Turn on Nimby
```

Comment:
Could we update to:

Turn on 'Not in my back yard' (NIMBY) to stop processing on the specified host

and

Turn off 'Not in my back yard' (NIMBY) to start processing on the specified host